admin_ui/backend/export_agents_yaml.py

```
@@ -0,0 +1,19 @@
+Usage: docker exec admin_ui python -m export_agents_yaml > contexts-recovered.yaml"""
+import json, sys, yaml
+from agents_store import AgentsStore
+
+def export_yaml(store: AgentsStore) -> str:
+    contexts = {}
+    for a in store.list_all():
+        ctx = {"provider": a["provider"], "prompt": a["prompt"]}
+        for k in ("voice", "greeting"):
+            if a[k]: ctx[k] = a[k]
+        if a["audio_profile"]: ctx["profile"] = a["audio_profile"]
+        if a["tools_json"]: ctx["tools"] = json.loads(a["tools_json"])
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)